pkgs/applications/audio/spotify-player/default.nix

```
@@ -8,6 +8,9 @@
 , dbus
 , fontconfig
 , libsixel
+, stdenv
+, darwin
+, makeWrapper
```

Can you please use `makeBinaryWrapper`, especially if you're looking to get it working on Darwin archs.
```suggestion
, makeBinaryWrapper
```